pkgs/rbrowser/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenvNoCC
+, makeDesktopItem
+, withBrave ? true
+, brave
+, withChromium ? true
+, chromium
+, withFirefox ? true
+, firefox
+, rofi
+, zsh
+}:
+let
+  inherit (stdenvNoCC.lib) optionalString;
+
+  desktopItem = makeDesktopItem {
+    categories = "GTK;Network;WebBrowser;";
+    desktopName = "Relay Browser";
+    exec = "rbrowser %U";
+    genericName = "Web Browser";
+    icon = "chromium";
+    mimeType = "x-scheme-handler/unknown;x-scheme-handler/about;text/html;text/xml;application/xhtml+xml;text/mml;x-scheme-handler/http;x-scheme-handler/https;";
+    name = "rbrowser";
+  };
+in
+stdenvNoCC.mkDerivation rec {
+  name = "rbrowser";
+
+  src = ./.;
+
+  phases = [ "installPhase" "fixupPhase" ];
+
+  installPhase = ''
+    install -Dm755 $src/bin/rbrowser $out/bin/rbrowser
+
+    substituteInPlace $out/bin/rbrowser \
+      ${optionalString withBrave "--subst-var-by brave_bin ${brave}/bin/brave"} \
```

Comment:
You're not really enabling brave here, you're only skipping setting the path to the brave binary. IMO bring the shell script inline so you can put the conditional around the entire brave-related sections of the code. Same applies to the rest.